plugins/ssl/ssl-certificate-expiry

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -u
 # -*- sh -*-
-# shellcheck disable=SC2039
+# shellcheck disable=SC2039,SC3043
```

Comment:
You do not need to add this exception, since the CI tests for `shellcheck` are executed with `-s dash`.